filebeat/input/log/input.go

```diff
@@ -93,7 +93,9 @@ func NewInput(
 	cleanupNeeded := true
 	cleanupIfNeeded := func(f func() error) {
 		if cleanupNeeded {
-			f()
+			if err := f(); err != nil {
+				logp.L().Named("input.log").Errorf("clean up function retruned an error: %w", err)
```

Comment:
You going to merge this suggession? Just a misspelling.

Reply:
Yes, I will. CI is not happy so I was delaying merging this change to avoid interrupting my running jobs.

Sorry, I should have mentioned that as soon as I saw your comment.